### PR TITLE
Fix gem vulnerabilities

### PR DESCRIPTION
**Pull request by isaacmg410**
Change rails version to 5.2.2.1 to fix vulnerabilities
Upgrade version

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,8 +5,9 @@ source "https://rubygems.org"
 ruby RUBY_VERSION
 
 # Specify your gem's dependencies in kaminari.gemspec
-gemspec
+gem 'rails', '5.2.2.1'
```
Comment (tramuntanal) [inline Gemfile:11]: Add a comment explaining why we add this and when it can be removed please.

File: Gemfile [hunk 2]
```diff
@@ -5,8 +5,9 @@ source "https://rubygems.org"
 ruby RUBY_VERSION
 
 # Specify your gem's dependencies in kaminari.gemspec
```
Comment (tramuntanal) [inline Gemfile]: Remove this comment?

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 2 above)
Comment (isaacmg410) [reply Gemfile]: Done

File: Gemfile (hunk 1 above)
Comment (isaacmg410) [reply Gemfile:11]: done

**Review by tramuntanal — APPROVED** [review]
Great job Isaac!

**Pull request merged**